src/ApiService/ApiService/onefuzzlib/NodeOperations.cs

```diff
@@ -470,12 +482,10 @@ public NodeOperations(ILogTracer log, IOnefuzzContext context)
     }
 
     public static string SearchStatesQuery(
```

Comment:
this was first bug - SearchStatesQuery would ignore current version node. We do not want that. (And Python code does not ignore it ether, so this fix is just aligning implementation with Python code)